Source/Scripts/upload-symbols-ios.sh

```diff
@@ -110,25 +121,6 @@ fi
 # Changed CFBundleName to CFBundleExecutable and Info.plist path
 PRODUCT_NAME=$(/usr/libexec/PlistBuddy -c "Print CFBundleExecutable" "${APP}/Info.plist")
 
-BUGSPLAT_SERVER_URL=$(/usr/libexec/PlistBuddy -c "Print BugsplatServerURL" "${APP}/Info.plist")
-BUGSPLAT_SERVER_URL=${BUGSPLAT_SERVER_URL%/}
-
-UPLOAD_URL="${BUGSPLAT_SERVER_URL}/post/plCrashReporter/symbol/"
-
 echo "App version: ${APP_VERSION}"
 
-UUID_CMD_OUT=$(xcrun dwarfdump --uuid "${APP}/${PRODUCT_NAME}")
-UUID_CMD_OUT=$([[ "${UUID_CMD_OUT}" =~ ^(UUID: )([0-9a-zA-Z\-]+) ]] && echo ${BASH_REMATCH[2]})
-echo "UUID found: ${UUID_CMD_OUT}"
-
-echo "Signing into bugsplat and storing session cookie for use in upload"
-
-COOKIEPATH="/tmp/bugsplat-cookie.txt"
-LOGIN_URL="${BUGSPLAT_SERVER_URL}/browse/login.php"
-echo "Login URL: ${LOGIN_URL}"
-rm "${COOKIEPATH}"
-curl -b "${COOKIEPATH}" -c "${COOKIEPATH}" --data-urlencode "currusername=${BUGSPLAT_USER}" --data-urlencode "currpasswd=${BUGSPLAT_PASS}" "${LOGIN_URL}"
-
-echo "Uploading ${FILE} to ${UPLOAD_URL}"
-
-curl -i -b "${COOKIEPATH}" -c "${COOKIEPATH}" -F filedata=@"${FILE}" -F appName="${PRODUCT_NAME}" -F appVer="${APP_VERSION}" -F buildId="${UUID_CMD_OUT}" $UPLOAD_URL
+$UPLOADER -b $BUGSPLAT_DATABASE -a $PRODUCT_NAME -v "$APP_VERSION" -f "**/*.{app,dSYM}" -d $UNZIP_LOCATION  -i $BUGSPLAT_CLIENT_ID -s $BUGSPLAT_CLIENT_SECRET
```

Comment:
Should the files in $UNZIP_LOCATION be cleaned uploading?

Reply:
Good point, I've added a command to remove those as well as the .zip file

Reply:
👍